packages/manager/modules/pci/src/projects/project/index.scss

```diff
@@ -0,0 +1,10 @@
+.pci-project-content {
+  .billing {
+    display: flex;
+    flex-direction: column;
+    .credit {
+      font-size: 32px;
+      font-weight: bold;
+    }
+  }
+}
```

Comment:
**nitpick:** Avoid using custom css.
`d-flex` and `flex-column` can be used. 
Ref: https://getbootstrap.com/docs/4.0/utilities/flex/

Reply:
Done.